README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="https://cloud.githubusercontent.com/assets/203457/24553916/1f3f88b6-162c-11e7-990a-731b2560f15c.png" alt="Guetzli" width="64"></p>
```

Comment:
Unfortunately, it can be only served with mime type "text/plain". There are third party services that fix that, but... they are third party.